CHANGELOG.md

```diff
@@ -24,6 +28,7 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 - Updates `Tabs` to be scrollable when the width of the tab set is greater than the viewport.
 - Updates the required comment field so it cannot be submitted when empty for the `FeedbackBox` component.
 - Updates the `Heading` component to use a custom `@media query` method to handle the responsive `font-size` styles.
+- Updates the `useNYPLBreakpoints` hook to include `isSmallerThanMedium` and `isSmallerThanDesktop`, both of which implement a desktop-first design approach.
```

Comment:
I'm pretty sure this was recently removed by Marty. Try updating this PR with the latest from `development`.